src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx

```diff
@@ -84,13 +85,13 @@ export const TransactionDetailsValueAndFeeSection: React.FC<Props> = ({ transact
                         }}
                       />
                     </View>
-                    {transaction.network !== Network.mainnet && <ChainBadge network={transaction.network} badgeYPosition={10} />}
+                    {transaction.network !== Network.mainnet && <ChainBadge chainId={transaction.chainId} badgeYPosition={10} />}
```

Comment:
did you mean `transaction.chainId !== ChainId.mainnet` instead of 1transaction.network !== Network.mainnet`